spec/requests/authentication_pages_spec.rb

```diff
@@ -39,4 +40,41 @@
       end
     end
   end
+
+  describe "authorization" do
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.